src/presentation/http/router/noteSettings.test.ts

```diff
@@ -343,6 +343,23 @@ describe('NoteSettings API', () => {
 
   describe('PATCH /note-settings/:notePublicId/team', () => {
     test('Update team member role by user id and note id, with status code 200', async () => {
+      // truncate tables which are needed
+      await global.db.query(`TRUNCATE public.notes, public.users, public.note_teams CASCADE`);
+
+      // restart autoincrement sequences for data to start with id 1
+      await global.db.query(`ALTER sequence users_id_seq RESTART WITH 1`);
+      await global.db.query(`ALTER sequence notes_id_seq RESTART WITH 1`);
+      await global.db.query(`ALTER sequence note_teams_id_seq RESTART WITH 1`);
+
+      // create test user (id 1)
+      await global.db.query(`INSERT INTO public.users ("email", "name", "created_at") VALUES ('testemal@CodeXmail.com', 'CodeX', CURRENT_DATE)`);
```

Comment:
same thing, maybe we should put it in a separate function? when you read the tests, you get lost in the huge amount of information